code-challenges.test.js

```diff
@@ -21,22 +48,73 @@ const numbersArray2 = [24, 27, 30, 33, 36]
 
 // b) Create the function that makes the test pass.
 
-// --------------------1) Create a function that takes a object as an argument and decides if the number inside it is evenly divisible by three or not.
-
-// a) Create a test with expect statements for each of the variables provided.
+  // PSEUDOCODE:
+    // declare a function called comboArray
+    // input: array of numbers, parameter called array
+    // output: an array with all the numbers multiplied by 3, same length as the original
+    // Process
+      // iterate across the values and multiply each value by 3 using the high order function map()
+      // return the array created by the high order function
+  
+  // FUNCTION
+  // const comboArray = ( array ) => {
+  //   return array.map((value) => {
+  //     return value * 3
+  //   })
+  // }
+  
+  // REFACTOR
+  const comboArray = ( array ) => {
+    return array.map(value =>value * 3)
+  }
+    // PASS
+    // PASS  ./code-challenges.test.js
+    // comboArray
+    //   ✓ takes in an array of numbers and returns an array with all the numbers multiplied by 3 (2 ms)
 
+// --------------------1) Create a function that takes a object as an argument and decides if the number inside it is evenly divisible by three or not.
 const object1 = { number: 15 }
 // Expected output: "15 is divisible by three"
 const object2 = { number: 0 }
 // Expected output: "0 is divisible by three"
 const object3 = { number: -7 }
 // Expected output: "-7 is not divisible by three"
 
+// a) Create a test with expect statements for each of the variables provided.
+describe("divNum", () => {
+    it("takes a object as an argument and decides if the number inside it is evenly divisible by three or not ", () => {
+    expect(divNum(object1.number)).toEqual("15 is divisible by three")
+    expect(divNum(object2.number)).toEqual("0 is divisible by three")
+    expect(divNum(object3.number)).toEqual("-7 is not divisible by three")
+    })
+})
+
+
 // b) Create the function that makes the test pass.
 
+// FUNCTION
+//create function named divnum
+//function takes 1 paramater, num
+//use a conditional and the modulo operator to get the correct output
+//run yarn jest
+const divNum = (value) => {
+    if (value % 3 === 0) {
+        return `${value} is divisible by three`
+    } else {
+        return `${value} is not divisible by three`
+    }
+}
+
```

Comment:
good structure. As stated on the test, the dot notation will be best suited for the logic in the function